src/Umbraco.Web.UI.Client/src/packages/tiptap/extensions/toolbar/media-picker.tiptap-toolbar-api.ts

```diff
@@ -99,11 +102,15 @@ export default class UmbTiptapToolbarMediaPickerToolbarExtensionApi extends UmbT
 		if (!media?.url) return;
 
 		const { width, height } = await imageSize(media.url, { maxWidth: this.maxWidth });
-		const src = await getProcessedImageUrl(media.url, { width, height, mode: ImageCropModeModel.MAX });
+		const { data } = await this.#imagingRepository.requestResizedItems([mediaUnique], {
+			width,
+			height,
+			mode: ImageCropModeModel.MAX,
+		});
 
 		const img = {
 			alt: media.altText,
-			src,
+			src: data[0]?.url ?? media.url,
 			'data-udi': `umb://media/${mediaUnique.replace(/-/g, '')}`,
 			width: width.toString(),
 			height: height.toString(),
```

Comment:
I'm concerned about the use of the width and height as they are set by the imageSize function for the original image and not for the resized image. Couldn't there be a discrepancy between the size, or is it more likely to happen if we didn't use the MAX mode?